brickflow_plugins/databricks/workflow_dependency_sensor.py

```diff
@@ -171,3 +171,92 @@ def execute(self):
 
             self.log.info(f"sleeping for: {self.poke_interval}")
             time.sleep(self.poke_interval)
+
+
+class WorkflowTaskDependencySensor(WorkflowDependencySensor):
```

Comment:
@maxim-mityutko can you please update the documentation for the same with example. Thanks!

Reply:
Done